### ISSUE TITLE
Gdpr compliance

### ISSUE THREAD

**Issue by staabm** [mnapoli/externals#74]
Doesnt the website miss a privacy page describing which external (non-Us etc) services are used (e.g. google fonts, analytics, ...)?

**Comment by mnapoli** [comment]
Yes that could be great to write, I don't have time at the moment to do it myself though.

**Comment by mnapoli** [comment]
No more google fonts, and analytics are privacy-compliant now, we should be good 👍

**Comment by staabm** [comment]
Awesome 😎
